tardis/apps/publication_workflow/components/Forms/SelectDatasetForm.jsx

```diff
@@ -0,0 +1,215 @@
+                <Form.Label>Select Datasets</Form.Label>
+                <div className="row">
+                  <div className="form-horizontal col-md-12">
+                    <div className="form-group has-search">
+                      {/* <span className="fa fa-search form-control-feedback"/> */}
+                      <input
+                        type="text"
+                        className="form-control"
+                        placeholder="Just start typing to filter datasets based on description"
+                        onKeyUp={onFilter}
+                      />
+                    </div>
+                  </div>
+                </div>
+                <Form.Control
+                  as="select"
+                  multiple
+                  onChange={handleDatasetSelectChange}
+                >
+                  {datasetList.map(value => (
+                    <option
+                      value={value.id}
+                      key={value.id}
+                    >
+                      {value.description}
+                    </option>
+                  )) }
+                </Form.Control>
+              </Form.Group>
+            </Col>
+            <Form.Control
+              as="input"
+              type="hidden"
+              name="selectedDatasets"
+              value={formik.values.selectedDatasets}
+              onChange={formik.handleChange}
+              isValid={formik.touched.selectedDatasets
+              && !formik.errors.selectedDatasets
+              }
+              isInvalid={!!formik.errors.selectedDatasets}
+            />
+            <Col className="col-md-6">
+              <h4>Added Datasets</h4>
+              <Table size="sm" responsive striped hover>
+                <thead>
+                  <tr>
+                    <th>Experiment</th>
+                    <th>Description</th>
+                    <th />
+                  </tr>
+                </thead>
+                <tbody>
+                  {formik.values.selectedDatasets.map(item => (
+                    <tr>
+                      <td>{item.experiment}</td>
+                      <td>{item.dataset.description}</td>
+                      <td>
+                        {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
+                        <span style={{ color: 'red', cursor: 'pointer' }} onClick={e => handleDatasetDelete(e, item)} role="button" tabIndex={0}>
+                          <i className="fa fa-trash" />
+                        </span>
+                      </td>
+                    </tr>
+                  ))}
+                </tbody>
+              </Table>
+            </Col>
+            <Form.Control.Feedback type="invalid" className="ml-3">
+              {formik.errors.selectedDatasets}
+            </Form.Control.Feedback>
+          </Row>
+          <Row>
+            <Form.Group />
+          </Row>
+        </Card.Body>
+      </Card>
+    </>
+  );
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
    );
```